### ISSUE TITLE
write section 4.5: utilized standards for milestone 3 report

### ISSUE THREAD

**Issue by ArdaYalcindag** [bounswe/bounswe2025group5#864]
### Task Description
Write the milestone report's section 4.5: Utilized Standards. The section should be written for both frontend, mobile and backend.

### Acceptance Criteria

- [x] WCAG 2.1 AA
- [x] Activity Streams 2.0
- [x] i18n

### Additional Details 
- [Link for the document](https://github.com/bounswe/bounswe2025group5/wiki/Customer-Milestone-3-Report-of-CMPE-451-Fall-2025#45-utilized-standards)

### Deadline
21.12.2025 - 8.00

**Comment by ArdaYalcindag** [comment]
WCAG 2.1 AA part is added to the document in [here](https://github.com/bounswe/bounswe2025group5/wiki/Customer-Milestone-3-Report-of-CMPE-451-Fall-2025#45-utilized-standards).

**Comment by bartukonca** [comment]
I have added the Activity Streams 2.0 part [here](https://github.com/bounswe/bounswe2025group5/wiki/Customer-Milestone-3-Report-of-CMPE-451-Fall-2025#4512-activitystreams-20)

**Comment by serdarbahar** [comment]
I have added the i18n part [here](https://github.com/bounswe/bounswe2025group5/wiki/Customer-Milestone-3-Report-of-CMPE-451-Fall-2025#4513-internationalization-i18n-standards)

**Comment by ArdaYalcindag** [comment]
All parts for the section 4.5 is completed. This issue can be closed now.
